README.md

```diff
@@ -126,6 +127,54 @@ client = Redis.new(semian: {
 })
 ```
 
+#### Net::HTTP
+For the `Net::HTTP` specific Semian adapter, since many external libraries may create
+HTTP connections on the user's behalf, the parameters are instead provided
+by calling specific functions in `Semian::NetHTTP`, perhaps in an initialization file.
+
+To give Semian parameters, assign a `proc` to `Semian::NetHTTP.semian_configuration`
+that takes two parameters, `host` and `port like `127.0.0.1` and `80` or `github_com` and `80`,
+and returns two values, a `semian_identifier` and a `Hash` with the keys as follows.
+
+```ruby
+SEMIAN_OPTIONS = { tickets: 1,
+                   success_threshold: 1,
+                   error_threshold: 3,
+                   error_timeout: 10 }
+Semian::NetHTTP.semian_configuration = proc do |host, port|
+  semian_identifier = "nethttp_#{host}_#{port}"
+  # Let's make it only active for github.com
+  if semian_identifier == "nethttp_github.com_80"
+    [semian_identifier, SEMIAN_OPTIONS]
+  else
+    [semian_identifier, nil]
+  end
+end
+
+# Called from within API:
+# semian_identifier, semian_options = Semian::NetHTTP.semian_configuration("github.com", 80)
+```
+
+The `semian_identifier` used as the name of the resource, and the `semian_options` are the
+configuration parameters. A return value of `nil` means Semian is disabled for that
```

Comment:
For ease of understanding, I'd explain what you mean by "resource" here. As in, something like: the `semian_identifier` specifies the name of the resource for which the `semian_options` apply. In practise this means that all endpoints that resolve to the same `semian_identifier` share their circuit breaker and bulkhead configuration. It could for sure be even more clear than that, play around with it a little.